packages/rss-handler/src/index.ts

```diff
@@ -114,15 +114,20 @@ export const rssHandler = Sentry.GCPFunction.wrapHttpFunction(
 
       // save each item in the feed
       for await (const item of feed.items) {
-        const publishedAt = item.pubDate || item.isoDate
-        console.log('Processing feed item', item.link, publishedAt)
+        const publishedAtString = item.pubDate || item.isoDate
```

Comment:
Do you know the format of this `pubDate`? Not sure if the RSS library parses it for us but i noticed its this format in my one testing feed `<pubDate>Wed, 12 Jul 2023 04:00:00 -0000</pubDate>`

Reply:
i think the `pubDate` format is not fixed. I could use the `isoDate` which is in ISO format that the RSS node library parsed from `pubDate`

Reply:
interesting, yeah i noticed the feed i have doesn't have isoDate, does the library add that for us?

Reply:
yeah, the `isoDate` is added by the library getting from `pubDate`